packages/storybook/stories/Examples/Layouts/snippets/ScreenShareLayout.snippet.tsx

```diff
@@ -13,8 +13,8 @@ const renderPersona = (props: PlaceholderProps): JSX.Element => (
 );
 
 const renderScreenSharePlaceholder = (): JSX.Element => (
-  <Stack className={mergeStyles({ height: '100%' })}>
-    <Stack verticalAlign="center" horizontalAlign="center" className={mergeStyles({ height: '100%' })}>
+  <Stack className={mergeStyles({ height: '100\u0025' })}>
```

Comment:
Is this intentional change and its to fix how it is shown in storybook docs? Totally fine if it is, just making sure, and surprised this works with the styles.

Reply:
saw this issue while tested so fixed it :)